libraries/testbot/index.js

```diff
@@ -1,46 +1,91 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// index.js is used to setup and configure your bot
+
+// Import required pckages
+const path = require('path');
 const restify = require('restify');
 
-const { BotFrameworkAdapter, MemoryStorage, UserState, ConversationState, InspectionState, InspectionMiddleware } = require('botbuilder');
+// Import required bot services. See https://aka.ms/bot-services to learn more about the different parts of a bot.
+const { BotFrameworkAdapter, MemoryStorage, ConversationState, UserState } = require('botbuilder');
+const { LuisHelper } = require('./dialogs/luisHelper');
+
+// Import the simple echo bot
 const { MyBot } = require('./bots/myBot')
 
+// This bot's main dialog.
+const { DialogAndWelcomeBot } = require('./bots/dialogAndWelcomeBot');
+const { MainDialog } = require('./dialogs/mainDialog');
+
+// Note: Ensure you have a .env file and include LuisAppId, LuisAPIKey and LuisAPIHostName.
+const ENV_FILE = path.join(__dirname, '.env');
+require('dotenv').config({ path: ENV_FILE });
+
+// Create adapter.
+// See https://aka.ms/about-bot-adapter to learn more about adapters.
 const adapter = new BotFrameworkAdapter({
     appId: process.env.MicrosoftAppId,
     appPassword: process.env.MicrosoftAppPassword
 });
 
-var memoryStorage = new MemoryStorage();
-var inspectionState = new InspectionState(memoryStorage);
-
-var userState = new UserState(memoryStorage);
-var conversationState = new ConversationState(memoryStorage);
-
-adapter.use(new InspectionMiddleware(inspectionState, userState, conversationState));
-
+// Catch-all for errors.
 adapter.onTurnError = async (context, error) => {
+    // This check writes out errors to console log
+    // NOTE: In production environment, you should consider logging this to Azure
+    //       application insights.
     console.error(`\n [onTurnError]: ${ error }`);
+    // Send a message to the user
```

Comment:
Set the Speak and InputHint properties in the activity (InputHint should be expecting input).

Reply:
this is done ^^